client/theme/theme.js

```diff
@@ -11,7 +13,34 @@ import gaps from './gaps';
 import borderRadius from './borderRadius';
 import typography from './typography';
 
-const theme = createMuiTheme({
+const isLightTheme = true;
+
+const lightTheme = createMuiTheme({
+  gaps,
+  borderRadius,
+  typography,
+  palette: {
+    type: 'light',
+    selected: { primary: 'rgba(129, 123, 123, 0.3)' },
+    primary: {
+      main: '#002449',
+    },
+    secondary: {
+      main: '#ffb100',
+    },
+    text: {
+      primary: '#002449',
+    },
+  },
+  header: {
```

Comment:
Please make a `sharedTheme` object that contains header, footer, and other shared components. Then the light and dark themes can add onto the `sharedTheme`.

Reply:
If I understand correctly, I'm not sure if a sharedTheme would allow me to accomplish what I had in mind. My approach was to separate the two themes into light and dark and utilize them in two different `<Themeprovider>` components that I wrap around the appropriate sections in `Route.jsx`. 

Since map utilizes dark and everything else utilizes light, ` <ThemeProvider theme={darkTheme}>` would wrap the map and `<ThemeProvider theme={lightTheme}>` would wrap everything else:
https://github.com/hackforla/311-data/blob/876e3bf4655790af25f7a349481bb9a70dfa0143/client/Routes.jsx#L26
https://github.com/hackforla/311-data/blob/876e3bf4655790af25f7a349481bb9a70dfa0143/client/Routes.jsx#L33

I'm open to trying out what you suggest once I understand a bit more. Perhaps we can discuss during the meeting. Thanks

Reply:
Addendum: Just updated the code so map utilizes dark and everything else utilizes the "default" theme

Reply:
Ah, sorry maybe `sharedTheme` isn't the right name. Something like `baseThemeItems`?

```
const baseThemeItems = {
  gaps,
  borderRadius,
  typography,
  header: {
    height: '62px',
  },
  footer: {
    height: '40px',
  },
}
```
Then, in `lightTheme` and `darkTheme`, you can destructure this.

Reply:
nice, this is a good suggestion. just got lost in translation. went ahead and grouped the repetitive items into a common object and referenced them from within the dark and light themes